Makefile

```diff
@@ -12,13 +12,14 @@ uninstall: kube-delete
 upgrade: kube-update
 
 docker-build:
+	build-plugin
```

Comment:
Would it be more conventional for this to appear after the colon on the previous line? This confused me at first.